### PR TITLE
[FIX] payment_stripe: incorrect flow mentionned in readme

### PR DESCRIPTION
**Pull request by chevalierv**

### CODE REVIEW

**Comment by chevalierv** [comment]
@robodoo r+

**Comment by AntoineVDV** [comment]
@robodoo r-

Could you fix the mention to Checkout Session too?

File: addons/payment_stripe/README.md [hunk 1]
```diff
@@ -6,16 +6,24 @@ SDK: [Stripe.js](https://stripe.com/docs/js) version `3`
...
+where the form is visible using a script tag.
+
+When the Web Elements needs to fetch/push information from/to Stripe or when a payment operation
+(e.g., refund, offline payment) is executed from the backend, a server-to-server API call is made to
+the appropriate API endpoint.
+
+This is achieved by following the [Collect payment details before creating an Intent](https://docs.stripe.com/payments/accept-a-payment-deferred) guide. It is preferred over the recommended "Payment Element" where payment intent is created
+beforehand because when the form is rendered, the `payment.transaction` object doesn't exist yet, so
+we don't have any reference to communicate to Stripe. Also, all the methods to create Stripe's
+object (like intents or customers) are on the `payment.transaction`​ object.
```
Comment (AntoineVDV) [inline addons/payment_stripe/README.md:20]:
```suggestion
we don't have any reference to communicate to Stripe. Also, all the methods to create Stripe
objects (e.g., intents or customers) are defined on the `payment.transaction`​ object.
```

File: addons/payment_stripe/views/payment_templates.xml [hunk 2]
```diff
@@ -15,7 +15,7 @@
 
     <template id="express_checkout" inherit_id="payment.express_checkout">
         <xpath expr="." position="inside">
-            <t t-call="payment_stripe.sdk_assets"/>
+            <t t-call="payment_stripe.sdk_assets"/>on
```
Comment (AntoineVDV) [inline addons/payment_stripe/views/payment_templates.xml:18]: https://tenor.com/view/shannon-sharpe-undisputed-nope-no-smh-gif-15099223

Comment (chevalierv) [reply addons/payment_stripe/views/payment_templates.xml]: :see_no_evil:

**Comment by chevalierv** [comment]
@robodoo r+

**Pull request closed without merging**